electron-app/src/components/accounts/FiatBalances.vue

```diff
@@ -52,13 +52,13 @@
                 <tr class="fiat-balances__total">
                   <td>Total</td>
                   <td></td>
-                  <td>
-                    {{
-                      fiatBalances.map(val => val.usdValue)
-                        | balanceSum
-                        | calculatePrice(exchangeRate(currency.ticker_symbol))
-                        | formatPrice(floatingPrecision)
-                    }}
+                  <td class="text-end">
+                    <amount-display
+                      fiat
+                      :value="
+                        fiatBalances.map(val => val.usdValue) | balanceSum
```

Comment:
Interesting, I had no idea that you could pipe to filters inside value assignments.